arraiw.properties

```
@@ -0,0 +1,2 @@
+version=0.87.0
```

Nit: bump to latest.
```suggestion
version=0.90.0
```

---

Done.